compose/ui/ui/src/desktopMain/kotlin/androidx/compose/ui/platform/AwtDragAndDropManager.desktop.kt

```diff
@@ -299,8 +299,9 @@ internal class ComposeTransferHandler(private val rootContainer: JComponent) : T
                 0,
                 false
             ),
-            // This seems to be ignored, and the initial action is MOVE regardless
-            MOVE
+            // This seems to be ignored, other than verifying that it's one of the flags returned in
+            // `getSourceActions`
+            transferData.supportedActions.first().awtAction
```

Comment:
```suggestion
            transferData.supportedActions.firstOrNull()?.awtAction ?: MOVE
```

Reply:
That won't work. The passed action *must* be one of the supported actions. `supportedActions` can't be empty.

Reply:
> supportedActions can't be empty.

We should add `require(supportedActions.isNotEmpty() { "..." }` then. Maybe to the constructor itself.